### ISSUE TITLE
无限问题

### ISSUE THREAD

**Issue by zcnew** [Rain-dew/YLCycleView#2]
这好像不能完全无限轮播，是伪无限吧，因为return (images?.count ?? 0) * 10000，若是10000设置小一点，会出现黑屏？(学习学习!)

**Comment by Rain-dew** [comment]
是的。这里其实是10000个，我自动滚动到中间，也就是5000张左右，开始就可以左右滚动，但是如果有个人很无聊滚5000张是会滚动到底的。但是我不认为会有这样无聊的人。当然。你可以这里设一个999999999999999.....

**Comment by Rain-dew** [comment]
设置小点是不会出现黑屏的。最后会滚动不了。其实你大可不必纠结此处

**Comment by zcnew** [comment]
好的，明白了，谢谢。
